testsuite/features/build_validation/init_clients/proxy.feature

```diff
@@ -52,7 +52,6 @@ Feature: Setup SUSE Manager proxy
     And file "/etc/sysconfig/apache2" should contain "rewrite" on "proxy"
     And file "/etc/sysconfig/apache2" should contain "version" on "proxy"
     And file "/etc/sysconfig/apache2" should contain "ssl" on "proxy"
-    And file "/etc/sysconfig/apache2" should contain "access_compat" on "proxy"
```

Comment:
Please if this step is a valid check in Uyuni, let's differ on the scenario using a label.

Reply:
I will recheck the spec file and adjust the code afterwards.

Reply:
It seems, in SUMA, we explicitly removed this. See https://github.com/SUSE/spacewalk/commit/73df9feac4563c6d6c1c2f45b279ae02d263baab
I can also no longer find it in the `spacewalk-proxy.spec` file on Uyuni. No idea what happened and why I saw it when I implemented the check.
We also removed it in Uyuni in 2019 according to the changelog: https://github.com/uyuni-project/uyuni/blob/089407703cafdc4b0a09659aadc53d42c53d8ac9/proxy/proxy/spacewalk-proxy.changes#L207

Reply:
See explanation below.